template.go

```diff
@@ -208,11 +214,15 @@ func executeAttrHtml(node *html.Node, field string, data map[string]interface{})
 
 // executeAttrIf executes the vue if attribute.
 func (vm *ViewModel) executeAttrIf(node *html.Node, field string, data map[string]interface{}) (*html.Node, bool) {
-	if value, ok := data[field]; ok {
-		if val, ok := value.(bool); ok && val {
-			return nil, false
-		}
+	negate := strings.HasPrefix(field, "!")
```

Comment:
Can you reply to this comment with a short-hand example of this case? If we keep it, then let's add a full working example later.